scss/grid/_grid.scss

```diff
@@ -30,6 +30,11 @@ $grid-column-align-edge: true !default;
 /// @type Number
 $block-grid-max: 8 !default;
 
+/// Selector used for an alias of column (with @extend). If `false`, no alias is created.
+/// @since 6.4.0
```

Comment:
I had add ```@since``` because the old version not have this variable and i think it's interesting to know the version this was introduce.